### PR TITLE
Add a validator

### PR DESCRIPTION
**Pull request by Devilla**
# Description*

- [x]  Added documentation for successfully adding a validator on Avalanche.

- [ ]  Create an ERC-20 Token using Avalanche C-Chain.

## Type of change*

Please delete options that are not relevant.

- [x] New feature (added screenshots and more description).
- [x] This documentation required an update.

## This PR created for* - 
- [x] DataHub Official Docs (Network Documentation, API docs etc.) changes/updates/fixes
- [x] Figment Learn Docs (Pathway, Tutorials, New Tutorials etc.) changes/updates/fixes

# Checklist*:

- [ ] I updated the `SUMMARY.md` file (FYI, It's needed only when you're adding/creating a new file like submitting a new tutorial, pathway section)
- [x] I checked and read the contribution guide before making this PR [**HERE**](https://docs.google.com/document/d/13LWLrWzZ34M0ldWGeDANcWxw9nEWk3AX3VwXRBIOs1M/edit)  
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Note - Above points marked with * are mandatory to fill :)

### CODE REVIEW

File: network-documentation/avalanche/tutorials/using-truffle-with-the-avalanche-c-chain.md [hunk 1]
```diff
@@ -16,28 +16,9 @@ You've created an [Avalanche DataHub](https://datahub.figment.io/sign_up?service
 
 ## Dependencies
```
Comment (yash-sharma1) [inline network-documentation/avalanche/tutorials/using-truffle-with-the-avalanche-c-chain.md:18]: @Devilla I don't think this content needs to be removed or change as this tutorial is from the official Avalanche Docs so we will prefer to keep it the same way. Any reason to change this :)
Can you please revert these changes so we can merge your PR after review

**Review by yash-sharma1 — CHANGES_REQUESTED** [review]
@Devilla If I'm not wrong you're adding 2-3 tutorials at once and as I see you made changes and mentioned only one addition of tutorial in the `SUMMARY.md` file 
So if you're adding more than one tutorial you need to add all of them in the `SUMMARY.md` file also this filename `network-documentation/avalanche/tutorials/Create an ERC-20 Token.md` should be hyphenized like `network-documentation/avalanche/tutorials/Create-an-ERC-20-Token.md` and then add it to the `SUMMARY.md` file
If all these tutorial are in series make them altogether in the group.

File: network-documentation/avalanche/tutorials/using-truffle-with-the-avalanche-c-chain.md (hunk 1 above)
Comment (Devilla) [reply network-documentation/avalanche/tutorials/using-truffle-with-the-avalanche-c-chain.md:18]: Okay, sure will revert these and update when Avalanche updates it.

**Comment by Devilla** [comment]
Too much conflicts here now will PR again with individual files. Thanks.

**Pull request closed without merging**